test/rootchain.js

```diff
@@ -299,16 +300,126 @@ contract('RootChain', async (accounts) => {
         await web3.currentProvider.send({jsonrpc: "2.0", method: "evm_increaseTime", params: [804800], id: 0});
         await web3.currentProvider.send({jsonrpc: "2.0", method: "evm_mine", params: [], id: 0});
         let currTime = (await web3.eth.getBlock(await web3.eth.blockNumber)).timestamp;
-        let diff = (currTime - oldTime) - 804800
+        let diff = (currTime - oldTime) - 804800;
         assert(diff < 3, "Block time was not fast forwarded by 1 week"); // 3 sec error for mining the next block
 
         // finalize
         let oldBal = (await rootchain.getBalance.call({from: accounts[2]})).toNumber();
-        await rootchain.finalizeExits({from: authority});
+        let oldFreeBalance = (await rootchain.getFreeBalance()).toNumber();
+        let exitresult = await rootchain.finalizeExits({from: authority});
```

Comment:
these types of rootchain calls do not have to be assigned to any variables

Reply:
same comment applies to all changes. This PR looks good to me otherwise though

Reply:
Not sure what you mean.

Reply:
the line can simply be. Similar to my other tests above
await rootchain.finalizeExits(...) 
instead of 
let exitresults = await ...

Reply:
Fixed